cmd/collectors/zapi/plugins/shelf/shelf.go

```diff
@@ -31,7 +31,7 @@ type shelfEnvironmentMetric struct {
 	currentSensor         map[string]float64
 }
 
-var eMetrics = []string{"power", "ambient_temperature", "max_temperature", "average_temperature", "average_fan_speed", "max_fan_speed", "min_fan_speed"}
+var eMetrics = []string{"power", "average_ambient_temperature", "min_ambient_temperature", "max_temperature", "average_temperature", "min_temperature", "average_fan_speed", "max_fan_speed", "min_fan_speed"}
```

Comment:
minor:we have enough of these that it probably makes sense to put each on a separate line and sort